feature-libs/quote/occ/adapters/occ-quote.adapter.ts

```diff
@@ -0,0 +1,229 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HttpClient } from '@angular/common/http';
+import { inject, Injectable } from '@angular/core';
+import {
+  QuoteAdapter,
+  QUOTE_ACTION_SERIALIZER,
+  QUOTE_COMMENT_SERIALIZER,
+  QUOTE_DISCOUNT_SERIALIZER,
+  QUOTE_LIST_NORMALIZER,
+  QUOTE_METADATA_SERIALIZER,
+  QUOTE_STARTER_SERIALIZER,
+  QUOTE_NORMALIZER,
+} from '@spartacus/quote/core';
+import {
+  Comment,
+  OccQuote,
+  Quote,
+  QuoteActionType,
+  QuoteDiscount,
+  QuoteList,
+  QuoteMetadata,
+  QuoteStarter,
+} from '@spartacus/quote/root';
+import {
+  ConverterService,
+  normalizeHttpError,
+  OccEndpointsService,
+  PaginationModel,
+} from '@spartacus/core';
+import { Observable, throwError } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+
+@Injectable()
+export class OccQuoteAdapter implements QuoteAdapter {
+  protected http = inject(HttpClient);
```

Comment:
@ChristophHi Rename this (and the following ones)

Reply:
Done!